docusaurus/src/css/custom.css

```diff
@@ -103,3 +103,11 @@ html[data-theme="dark"] .navbar.navbar .navbar-sidebar .navbar__brand .navbar__l
         max-width: 38px;
     }
 }
+
+/* ==========================================================================
+   Sidebar tweaks
+   ========================================================================== */
+
+html[data-theme="dark"] [class^="sidebarLogo_"] > img {
```

Comment:
This approach should work now but it can still break and go unnoticed if the class name changes in the future. The correct solution is to configure the sidebar as [designed](https://v2.docusaurus.io/docs/next/theme-classic/#navbar-title--logo) by providing a different logo for dark mode.

Reply:
We had a similar issue before with the header icon and I guess that Docusaurus didn't support this `srcDark` logo. I also think this is time design a logo for the dark theme and update accordingly. What do you think @satazor?

Reply:
I think we can land this fix for now, and later design a logo that plays well on light and dark backgrounds.